### PR TITLE
add missing braces

### PR DESCRIPTION
**Pull request by grimreaper**

### CODE REVIEW

File: code/src/java/plugin/overland/model/TravelMethodFactory.java [hunk 1]
```diff
@@ -104,8 +104,7 @@
 						TravelMethod tm = TravelMethodFactory.create(methodSet);
 						tms.add(tm);
 					}
-				}
-				catch (Exception e)
+				} catch (Exception e)
```
Comment (thpr) [inline code/src/java/plugin/overland/model/TravelMethodFactory.java]: Braces are on wrong line

File: code/src/java/plugin/overland/model/TravelMethodImplementation.java [hunk 2]
```diff
@@ -740,7 +767,10 @@ public int getSize()
 		public Object getElementAt(int index)
 		{
 			if (selectedMethod == null || !listByWay.containsKey(selectedMethod.getWay()))
+
```
Comment (thpr) [inline code/src/java/plugin/overland/model/TravelMethodImplementation.java]: no gap please

File: code/src/java/plugin/overland/model/TravelMethodImplementation.java [hunk 3]
```diff
@@ -802,10 +839,16 @@ private void fireMethodChanged(MethodModel source, Method previousMethod)
 				end = Math.min(end, previousSize);
 				// increment start for each identical element at the start of the paces lists
 				for (int i = 0; i < end && previousList.get(i).equals(selectedList.get(i)); i++, start++)
+
```
Comment (thpr) [inline code/src/java/plugin/overland/model/TravelMethodImplementation.java]: gap

File: code/src/java/plugin/overland/model/TravelMethodImplementation.java [hunk 4]
```diff
@@ -802,10 +839,16 @@ private void fireMethodChanged(MethodModel source, Method previousMethod)
...
+				{
 					;
+				}
 				// decrement end for each identical element at the end of the paces lists
 				for (int i = end - 1; i > start && previousList.get(i).equals(selectedList.get(i)); i--, end--)
+
```
Comment (thpr) [inline code/src/java/plugin/overland/model/TravelMethodImplementation.java]: gap

File: code/src/java/plugin/overland/model/TravelMethodImplementation.java [hunk 5]
```diff
@@ -851,7 +896,10 @@ public int getSize()
 		public Object getElementAt(int index)
 		{
 			if (selectedMethod == null || index < 0 || getList(selectedMethod).size() <= index)
+
```
Comment (thpr) [inline code/src/java/plugin/overland/model/TravelMethodImplementation.java]: gap

File: code/src/java/plugin/overland/model/TravelMethodImplementation.java [hunk 6]
```diff
@@ -906,10 +959,16 @@ protected void fireMethodChanged(MethodModel source, Method previousMethod)
 				end = Math.min(end, previousSize);
 				// increment start for each identical element at the start of the paces lists
 				for (int i = 0; i < end && previousList.get(i).equals(selectedList.get(i)); i++, start++)
+
```
Comment (thpr) [inline code/src/java/plugin/overland/model/TravelMethodImplementation.java]: gap

File: code/src/java/plugin/overland/model/TravelMethodImplementation.java [hunk 7]
```diff
@@ -906,10 +959,16 @@ protected void fireMethodChanged(MethodModel source, Method previousMethod)
...
+				{
 					;
+				}
 				// decrement end for each identical element at the end of the paces lists
 				for (int i = end - 1; i > start && previousList.get(i).equals(selectedList.get(i)); i--, end--)
+
```
Comment (thpr) [inline code/src/java/plugin/overland/model/TravelMethodImplementation.java]: gap

File: code/src/java/translation/util/Tips.java [hunk 8]
```diff
@@ -312,15 +326,16 @@ private static void generateTips(File rootDirectory,
 		finally
 		{
 			if (translationReader != null)
+			{
 				try
 				{
 					translationReader.close();
-				}
-				catch (IOException e)
+				} catch (IOException e)
```
Comment (thpr) [inline code/src/java/translation/util/Tips.java]: brace

**Comment by grimreaper** [comment]
think I fixed them all

**Pull request merged**